addOns/client/src/main/resources/org/zaproxy/addon/client/resources/spiderClient-max.yaml

```diff
@@ -0,0 +1,14 @@
+  - type: spiderClient                 # The client spider - a new spider which explores modern web apps more effectively
+    parameters:
+      context:                         # String: Name of the context to spider, default: first context
+      user:                            # String: An optional user to use for authentication, must be defined in the env
+      url:                             # String: Url to start spidering from, default: first context URL
+      maxDuration:                     # Int: The max time in minutes the spider will be allowed to run for, default: 0 unlimited
+      maxCrawlDepth:                   # Int: The maximum tree depth to explore, default 5
+      maxChildren:                     # Int: The maximum number of children to add to each node in the tree
+      numberOfBrowsers:                # Int: The number of browsers the spider will use, more will be faster but will use up more memory
+      inScopeOnly:                     # Boolean: If true then any URLs requested which are out of scope will be ignored, default: true
```

Comment:
We're supporting scope via AF but not GUI?

Reply:
We should add this to the GUI 😁

Reply:
Was this meant to be "subtree only"? "In scope" is not implemented in the client spider (though we could add that as well).